### PR TITLE
API: Address | create group api addresses

### PR DESCRIPTION
**Pull request by Sotatek-GiangNguyen7**

### CODE REVIEW

**Comment by Sotatek-GiangNguyen7** [comment]
https://sotatek-davidnguyen4.github.io/cardano-api-tests/91/
image

File: src/test/java/tests/addresses/TopAddressTests.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+        topAddressSteps.verifyAttributeIsSortedCorrectly(topAddressModel);
+
+        //with page = 2 and size = 3 sort = id
+        param = new CreateParameters().withPage(page).withPageSize(pageSize).withSort(sort).getParamsMap();
+        topAddressSteps
+                .getDataForTopAddress(param)
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+
+        topAddressSteps.verifyAttributeIsSortedCorrectly(topAddressModel)
```
Comment (ghost) [inline src/test/java/tests/addresses/TopAddressTests.java:64]: this topAddressModel is reponse of param = new CreateParameters().withSort(sort).getParamsMap();

File: src/test/java/tests/addresses/TopAddressTests.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                        .verifyDataAmountIsCorrect(pageSize, topAddressModel.getTotalItems(), topAddressModel.getData().size());
```
Comment (ghost) [inline src/test/java/tests/addresses/TopAddressTests.java]: i think miss verify sort by id

File: src/test/java/tests/addresses/TopAddressTests.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+        //with page = blank and size = 3 sort = balance
+        param = new CreateParameters().withPageSize(pageSize).withSort(sort).getParamsMap();
+        topAddressSteps
+                .getDataForTopAddress(param)
+                .validateStatusCode(HttpURLConnection.HTTP_OK);
+
+        topAddressSteps.verifyAttributeIsSortedCorrectly(topAddressModel);
```
Comment (ghost) [inline src/test/java/tests/addresses/TopAddressTests.java]: cmt like above

File: src/test/java/tests/addresses/TopAddressTests.java (hunk 2 above)
Comment (Sotatek-GiangNguyen7) [reply src/test/java/tests/addresses/TopAddressTests.java]: i don't miss cause my comment is wrong just have sort = "balance"

**Comment by Sotatek-GiangNguyen7** [comment]
image

**Review by ghost — APPROVED** [review]

**Pull request merged**